frontend/src/components/TodoList.tsx

```diff
@@ -0,0 +1,19 @@
+import React from "react";
+import { Todo } from "../models/todo.model";
+
+import TodoListItem from "./TodoListItem";
+
+export default function TodoList(props: any) {
+  console.log(props.todos);
```

Comment:
Do you know about the expected type of props? if yes then replace any with a model or type